src/scripts/cfo.zig

```diff
@@ -467,11 +502,79 @@ fn run_fuzzers_prepare_tasks(shell: *Shell, gh_token: ?[]const u8) !Tasks {
         }
     }
 
+    const runtimes = try shell.arena.allocator().alloc(u64, working_directory.items.len);
+    @memset(runtimes, 1);
+
+    var map = Tasks.Map.init(shell.arena.allocator());
+    for (seed_template.items, 0..) |template, i| {
+        try map.putNoClobber(.{
+            .fuzzer = template.fuzzer,
+            .commit = template.commit_sha,
+        }, i);
+    }
+
     return .{
+        .map = map,
         .working_directory = working_directory.items,
         .seed_template = seed_template.items,
         .weight = weight,
+        .runtimes = runtimes,
+    };
+}
+
+/// Initialize the runtime of each task in `tasks`.
+fn run_fuzzers_prepare_tasks_runtime(tasks: Tasks, tasks_cache: Tasks) void {
```

Comment:
I am starting to hate my convention of prefixing everything with `run_fuzzers` 🤣